### PR TITLE
feat: public holidays command

### PR DESCRIPTION
**Pull request by NathanInnes**
## Description

Adds a new command that lists upcoming worldwide public holidays for the next 7 days.

NOTE: Response limited to 5 due to limitations with discord message length.

## Test Results
image

### Example of error handling
image

### CODE REVIEW

File: src/commands/utils/holidays.ts [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+                        inline: true,
+                    },
+                ]),
+            });
+            return msg.channel.send({ embeds: [holidaysEmbed] });
+        } catch (err) {
+            const errorEmbed = makeEmbed({
+                title: 'Error | Public Holidays',
+                description: `An error occurred ${err}`,
+                color: 'RED',
```
Comment (NathanInnes) [inline src/commands/utils/holidays.ts:43]: Awaiting #289 to be merged so line 43 can be changed.

**Comment by benw202** [comment]
Could make pages scrollable? Like in .help if you wanted to display more. If not, looks great!

**Comment by NathanInnes** [comment]
Could do

**Comment by benw202** [comment]
Can this be limited to #bot-commands please :)

**Comment by NathanInnes** [comment]
> While I appreciate the efforts this isn't aviation, simming nor fbw related and don't see the added benefit it provides to the discord server.
> 
> 
> 
> IMO a bot that outlines VATSIM events occuring in the next 7 days would be more appropriate (assuming the API exists), or something along those lines that can tangibly benefit the server.

You have a fair point. I can look into changing this into an events list command across networks. Thanks for input.

**Comment by NathanInnes** [comment]
Closing for new PR with for VATSIM events.

**Pull request closed without merging**